scripts/test/shared.py

```diff
@@ -52,6 +52,12 @@ def parse_args(args):
               ' Default: bin/ of current directory (i.e. assume an in-tree build).'
               ' If not specified, the environment variable BINARYEN_ROOT= can also'
               ' be used to adjust this.'))
+    parser.add_argument(
+        '--binaryen-lib', dest='binaryen_lib', default='',
+        help=('Specifies a path to where the built Binaryen shared library resides at.'
+              ' Default: lib/ of current directory (i.e. assume an in-tree build).'
+              ' If not specified, the environment variable BINARYEN_ROOT= can also'
+              ' be used to adjust this.'))
```

Comment:
Can this default to ../lib relative to binaryen-bin so I don't need to always specify both for default installations?

Reply:
Will do, thanks for review.